src/key.rs

```diff
@@ -122,6 +122,17 @@ impl SecretKey {
         PublicKey::from_slice(&uncompressed[1..]).expect("The length of the key is correct; qed")
     }
 
+    /// Export stored, unencrypted, plain private key, use with caution
+    /// Do not expose this key in logs, etc. Use only if needed
+    #[cfg(feature = "export-private-key")]
+    pub fn private(&self) -> [u8; 32] {
```

Comment:
```suggestion
    pub fn raw(&self) -> [u8; 32] {
```

WDYT about calling the function `raw` instead of `private`? I think it makes sense as it returns the raw key.